launcher/cog-launcher.c

```diff
@@ -388,6 +398,13 @@ cog_launcher_startup(GApplication *application)
         g_object_set(self->shell, "config-file", key_file, NULL);
     }
 
+#if COG_USE_WPE2
+    if (!self->automated) {
+        webkit_network_session_set_tls_errors_policy(self->network_session,
+                                                     s_options.ignore_tls_errors ? WEBKIT_TLS_ERRORS_POLICY_IGNORE
+                                                                                 : WEBKIT_TLS_ERRORS_POLICY_FAIL);
+    }
+#else
 #if WEBKIT_CHECK_VERSION(2, 32, 0)
```

Comment:
I would replace both lines with an `#elif`:

```suggestion
#elif WEBKIT_CHECK_VERSION(2, 32, 0)
```